Ground_RealSense_Height: 1.05
Ground_Neck_Height: 0.95
Ground_Arm_Height: 0.78 # 0.48
Shoulder_Elbow_Length: 0.22
Elbow_Wrist_Length: 0.17
Wrist_Endeffector_Length: 0.17

#Gen2
Grasp_Torque_Thres: 30
Origin_Angle: [880, 3100, 2080, 2050, 700, 1100]  #[840, 3260, 2080, 2150, 700, 2000]
Origin_Pose: [0,0,0]
Carry_Pose: [-85, 90, 90]
Recieve_Pose: [-40, 70, -30, false]
Give_Pose: [-35, 75, -35, true]
Point_Pose: [-85, 90, 0]
Look_Floor_Pose: [-65, 0, 20]

Start_Pose: "Look_Floor_Pose"
#Gen3
#Origin_Angle: [830, 3336, 2065, 2065, 2150, 1794]
Gear_Ratio: [2.1, 2.1, 1, 1, 1, 1]